bin/mvn-search

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env node
+const yargs = require('yargs');
+
+const args = yargs
+    .scriptName('mvn-search')
+    .usage('$0 <query-string> [args]', 'search for maven dependency', (yargs) => {
```

Comment:
i think it would be better to put the options before the query string. I mean `$0 [args] <query-string>`.

Reply:
yeah, done.

Reply:
Speaking about the options. I was thinking why not use [minimist](https://www.npmjs.com/package/minimist). This will have `_[0]` as Search Term and options can be passed in any fashion. Please let me know if you think of anything else.